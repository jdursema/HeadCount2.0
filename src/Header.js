import React from 'react';
import './header.css';


const Header = () => {
  return (
     <div className="header">
      <h1>Head Count 2.0</h1>
     </div>
    )
}

export default Header;
