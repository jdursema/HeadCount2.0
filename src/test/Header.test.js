import React from 'react';
import { shallow, mount } from 'enzyme';
import Header from '../Header';

describe('Header Test', () => {

  it('should be defined', () => {
    const header = shallow(<Header />);

    expect(header).toBeDefined();
  });

  it('should match snap shot', () => {
    const header = shallow(<Header />);

    expect(header).toMatchSnapshot();
  });

});
