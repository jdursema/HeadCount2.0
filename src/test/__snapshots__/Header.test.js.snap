exports[`Header Test should match snap shot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Head Count 2.0
</h1>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Head Count 2.0",
      },
      "ref": null,
      "rendered": "Head Count 2.0",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Head Count 2.0
</h1>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Head Count 2.0",
        },
        "ref": null,
        "rendered": "Head Count 2.0",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
